backend/director/tools/videodb_tool.py

```
@@ -489,34 +490,37 @@ def _download_video_file(self, video_url: str, save_at: str) -> bool:
 
-        if not response.headers.get('Content-Type', '').startswith('video'):
+        if not response.headers.get("Content-Type", "").startswith("video"):
             raise ValueError(f"The URL does not point to a video file: {video_url}")
 
-        with open(save_at, 'wb') as file:
+        with open(save_at, "wb") as file:
             file.write(response.content)
 
-    def text_to_video(self, prompt: str, save_at: str, duration: float, config: dict = {}):
+    def text_to_video(
+        self, prompt: str, save_at: str, duration: float, config: dict = {}
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)